src/views/Layout.jsx

```diff
@@ -26,10 +26,16 @@ export function Layout() {
 
 					{currentToken ? (
 						<>
-							<NavLink to="/list" className="Nav-link">
+							<NavLink
+								to="/list"
+								className="Nav-link bg-orange-700  border border-black rounded-[3px] text-white w-[81px] lg:w-[122px] h-[35px] lg:h-[76px] text-[14px] lg:text-[20px] lg:leading-[50px] font-bold font-poppins"
+							>
 								List
 							</NavLink>
-							<NavLink to="/add-item" className="Nav-link">
+							<NavLink
+								to="/add-item"
+								className="Nav-link bg-[#0CC296] border border-black rounded-[3px] text-white  w-[81px] lg:w-[122.35px]  h-[35px] lg:h-[76px] text-[14px] lg:text-[20px] lg:leading-[50px] font-bold font-poppins"
```

Comment:
I think for buttons it's best to do padding rather than width and height. This is how it ends up looking on my browser in desktop view
![image](https://user-images.githubusercontent.com/74952593/203810891-bfc5509c-45c1-4c9b-8059-8f18f05b4a29.png)

This is browser view. It's okay but I think the spacing around the text is off. That's why I think setting padding instead might work best.
![image](https://user-images.githubusercontent.com/74952593/203811046-39adf723-19b4-4eb1-b3f0-9dad98e9887f.png)